src/app/templates/app/submit_new_license.html

```diff
@@ -612,6 +614,27 @@
     });
   }
 </script>
+<script type="text/javascript">
+  // Add an event listener to the "Check License Text" button
+  document.getElementById('check-license-btn').addEventListener('click', function() {
+    // Get the license text from the textarea
+    const licenseText = document.getElementById('{{ form.text.id_for_label }}').value;
+    var csrftoken = $('input[name=csrfmiddlewaretoken]').val();
+    var dict1 = {
+      url: '/app/diff/',
+      type: 'POST',
+      data: {licensetext: licenseText},
+      headers: {
+        'X-CSRFToken': csrftoken
+      }
+    }
+    doRequest(dict1);
```

Comment:
nit: Better naming can be done `checkLicenseDiff`

Reply:
`doRequest` is the in-built method which I have used, are you suggesting it for that or something else?